src/quadpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -73,3 +125,23 @@ def process(self, instance):
 
         self.log.info("Extracted instance '%s' to: %s",
                        instance.name, representation)
+
+    @staticmethod
+    def is_shot():
```

Comment:
same as in the `load_blend` file: seems not generic that the condition for shot is that there is at last one _in the asset name

Reply:
also maybe move this util function to the api or an utils file